packages/@biomejs/js-api/tests/lintContent.test.ts

```diff
@@ -13,11 +18,60 @@ describe("Biome WebAssembly lintContent", () => {
 		biome.shutdown();
 	});
 
-	it("should lint content", () => {
-		const result = biome.lintContent("if (a == b) {}", {
-			filePath: "example.js",
+	describe("fixFileMode is undefined/omitted", () => {
+		it("should emit diagnotics", () => {
+			const result = biome.lintContent(inputCode, {
+				filePath: "example.js",
+			});
+			expect(result.diagnostics).toMatchObject([
```

Comment:
I've changed the assertions on diagnostics to just check the category instead of using snapshots - I felt this was more maintainable, but happy to change back if you disagree